sdk/storage/azure-storage-blob/src/main/java/com/azure/android/storage/blob/models/BlobGetTagsHeaders.java

```diff
@@ -0,0 +1,4 @@
+package com.azure.android.storage.blob.models;
+
+public class BlobGetTagsHeaders {
+}
```

Comment:
Why is this empty?

Reply:
@rickle-msft Filled it in, Can you take another look?